### ISSUE TITLE
X-rays files are deleted

### ISSUE THREAD

**Issue by HeyySuraj** [junaidiqbalsyed/Covid_detection_CNN#3]
I came across the problem where x-rays files are not there could u plz help me with that

**Comment by AchuAshwath** [comment]
> 
> 
> I came across the problem where x-rays files are not there could u plz help me with that

https://www.kaggle.com/tawsifurrahman/covid19-radiography-database

this might help!

**Comment by junaidiqbalsyed** [comment]
Yes, you can use this data set. As covid is a type of Phenemonia you can edit the data set

**Comment by HeyySuraj** [comment]
thanks for sharing this data set I really need It

**Comment by samm106** [comment]
> Yes, you can use this data set. As covid is a type of Phenemonia you can edit the data set

should i only use covid and normal type images from this  dataset
???
